### PR TITLE
RFC 6961 changes for TLS 1.2 vs TLS 1.3

### PR DESCRIPTION
**Pull request by paulehoffman**

### CODE REVIEW

**Comment by mcmanus** [comment]
hmm. we do need to fix this - but the proposed fix removes a normative should and introduces tls 1.3 which creates a publishing dependency. The dep is rather minor, but the benefit to DoH even more minor.

let me see if I can come up with a fix that is just a SHOULD inline/staple

**Comment by paulehoffman** [comment]
We cannot say SHOULD use 6961 because that is not allowed in TLS 1.3. I purposely didn't make a publishing dependency to TLS 1.3 by naming it but not giving a link; every reader of this document will know what we are talking about.

File: draft-ietf-doh-dns-over-https-latest.mkd [hunk 1]
```diff
@@ -536,9 +536,10 @@ revocation status of the certificates being used by TLS. If this check
 is done as part of the DNS API server connection procedure and the
 check itself requires DNS resolution to connect to the third party a
 deadlock can occur. The use of an OCSP {{RFC6960}} server is one
-example of how this can happen. DNS API servers SHOULD utilize OCSP
-Stapling {{RFC6961}} to provide the client with certificate revocation
-information that does not require contacting a third party.
+example of how this can happen.
+Differenet versions of TLS use different mechanisms to get OCSP
```
Comment (chantra) [inline draft-ietf-doh-dns-over-https-latest.mkd]: s/differenet/different/

**Comment by mcmanus** [comment]
I understand the problem. We can probably say something like SHOULD
generically staple or inline (such as 6961 where it applies). tls 1.3 does
do stapling, it just doesn't do it via 6961.

**Comment by mcmanus** [comment]
wdyt of that @paulehoffman ?

**Comment by paulehoffman** [comment]
Reworded again, and back to you @mcmanus

**Pull request merged**